js_modules/dagster-ui/packages/ui-core/src/schedules/ScheduleRoot.tsx

```diff
@@ -133,7 +133,7 @@ export const ScheduleRoot = (props: Props) => {
                 tabs={tabs}
               />
             ) : (
-              <RunsFeedTableWithFilters filter={runsFilter} />
+              <RunsFeedTableWithFilters filter={runsFilter} actionBarComponents={tabs} />
```

Comment:
This was causing these tabs to disappear when you switched to Run History

<img width="359" alt="image" src="https://github.com/user-attachments/assets/0cdff671-93ce-4f3c-9f20-43e2d4ad95bd">